src/mmw/js/src/modeling/models.js

```diff
@@ -557,23 +557,63 @@ var ProjectModel = Backbone.Model.extend({
                         contents: at.getResultCSV(),
                     };
                 }),
+            scenarios = self.get('scenarios'),
+            lowerAndHyphenate = function(name) {
+                    return name.toLowerCase().replace(/\s/g, '-');
+                },
+            isTR55 = self.get('model_package') === utils.TR55_PACKAGE,
```

Comment:
Consider instead adding helper functions like `createTR55ModelFileCSVs()` and `createMapshedModelFileCSVs()`, so that we don't have everything `isTR55 ?`ed. It doesn't seem like the duplication would be too bad.

Reply:
Where were you envisioning these helper methods would go? Within `exportToHydroShare`, or within `modelFiles`, or within `ResultModel` or `ResultCollection`?

Reply:
I was picturing within `exportToHydroShare`, though the `ResultCollection` might be a good idea too...it only needs to know the scenarios and the model package?

Reply:
Added in 21e453e.